### ISSUE TITLE
Format paper using LaTeX

### ISSUE THREAD

**Issue by NinoDS** [sphynx-owner/JFA_driven_motion_blur_demo#3]
I very much enjoyed reading your paper/writeup and found your approach very interesting, even though I have limited knowledge on the subject. I believe though that using a formatting language like LaTeX could enhance the presentation of (mathematical) content and improve readability.

I have reformatted your paper as an example, which you can view [here](https://github.com/user-attachments/files/16150727/latexreport.pdf). If you're interested, I'd be happy to share the LaTeX file with you.

**Comment by sphynx-owner** [comment]
Thank you for that. It does seem to elevate the paper quite a bit. Sadly, It needs quite a bit more work as the code has changed since and I got some feedback about my lack of details which I would like to iterate upon. I would love to have the LaTeX file so I can modify it when I update the paper.

**Comment by NinoDS** [comment]
Of course, [here](https://github.com/user-attachments/files/16162094/paper.zip) is the full paper including images and bib.

**Comment by sphynx-owner** [comment]
fixed with latest commit

**Comment by sphynx-owner** [comment]
wrong issue
